db/description/schema.go

```diff
@@ -161,6 +203,47 @@ func GetSchemas(
 	return descriptions, nil
 }
 
+// GetSchemas returns all schema versions in the system.
+func GetAllSchemas(
+	ctx context.Context,
+	txn datastore.Txn,
+) ([]client.SchemaDescription, error) {
+	prefix := core.NewSchemaVersionKey("")
+	q, err := txn.Systemstore().Query(ctx, query.Query{
```

Comment:
suggestion: I think you should use `datastore.DeserializePrefix[client.SchemaDescription]` for this

Reply:
I only recently noticed that in one of your PRs and then forgot about it, will have a look, but there is a chance I'll want to leave it as is, as I might feel that calling that function hides too much (as the purpose of this file is to show all that stuff in one place).

- [ ] Checkout datastore.DeserializePrefix

Reply:
It would be just one click away from visibility though so I'm not sure the duplication would be worth it.

Reply:
It is not just about that, but extracting this function to another shared function creates an oddity in this file, which itself hinders readability in some cases.

Worse though is that it inhibits change, if this func needs to change slightly (for whatever reason), it is currently very easy to do so - you just change it, however if it is calling `datastore.DeserializePrefix`, then either you have to risk serious damage by changing that function, or overcome a physcological barrier and un-refactor it and then make the change - that physcological barrier can be non-trivial in some cases and is frequent cause of large bloated shared functions.

Reply:
I had a look, and I'm still more comfy with the independent implementations.  Will be leaving as is.